static/app/components/button.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import {Fragment, useState} from 'react';
+
+import {Button} from 'sentry/components/button';
+import Matrix from 'sentry/components/stories/matrix';
+import {IconDelete} from 'sentry/icons';
+import storyBook from 'sentry/stories/storyBook';
+
+export default storyBook('Button', story => {
+  const sizes = ['md' as const, 'sm' as const, 'xs' as const, 'zero' as const];
+  const priorities = [
```

Comment:
Why can't the whole array just be `as const`?

Reply:
I don't remember. can you test it out and lmk what typescript says?